test_scripts/API/VehicleData/Refactor_Fuel_Information_Related_Vehicle_Data/006_OnVehicleData_with_only_one_param_in_fuelRange_structure.lua

```diff
@@ -0,0 +1,40 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0256-Refactor-Fuel-Information-Related-Vehicle-Data.md
+-- Description: SDL successfully processes OnVehicleData notification and transfers
+-- it to a mobile app if HMI sends only one param in FuelRange structure
+-- In case:
+-- 1) App is subscribed to get `FuelRange` data
+-- 2) HMI sends valid OnVehicleData notification of `FuelRange` structure with only one param
+-- SDL does:
+-- 1) process this notification and transfer it to mobile
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/VehicleData/Refactor_Fuel_Information_Related_Vehicle_Data/common')
+
+-- [[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+local params = {
+  fuelRange = {
+    dataType = "VEHICLEDATA_FUELRANGE",
+    resultCode = "SUCCESS"
+  }
+}
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("Register App", common.registerApp)
+runner.Step("PTU", common.policyTableUpdate, { common.pTUpdateFunc })
+runner.Step("Activate App", common.activateApp)
+runner.Step("App subscribed to fuelRange param", common.rpcVehicleData, {"SubscribeVehicleData", "fuelRange", params, "SUCCESS", true})
+
+runner.Title("Test")
+for k,v in pairs(common.sendsOneParams) do
```

Comment:
Please use the table with all parameters from common file and remove the table with separated parameters from common